share/spice/zomato/zomato.js

```diff
@@ -0,0 +1,80 @@
+(function (env) {
+    "use strict";
+    var query_orig = DDG.get_query();
+    //$.getJSON('https://www.zomato.com/duckduckgo/ddg_track?value='+encodeURIComponent(query_orig));
+    env.ddg_spice_zomato = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('zomato');
+        }
+        //console.log(api_result);
+        Spice.add({
+            id: "zomato",
+            name: "Places",
+            model: "Place",
+            view: "Places",
+            signal: "high",
+            data: api_result.restaurants,
+            meta: {
+                sourceName: "Zomato",
+                sourceUrl: api_result.more_at,
+                snippetChars: 110,
+                sourceIcon: true,
+                sourceIconurl: 'https://www.zomato.com/images/logo/zomato_favicon3.png',
+                itemType: 'Places',
+                searchTerm: query_orig
+            },
+            templates: {
+                group: 'places',
+                options: {
+                    moreAt: true
+                }
+            },
+            relevancy: {
+                dup: 'url'
+            },
+            /*
+            sort_fields: {
```

Comment:
@moollaza I just realised that these `sort_fields` are not as a sort option for user, instead there are sorted on these rules and displayed, right?